mantis-common/src/main/java/io/mantisrx/server/core/ServiceRegistry.java

```diff
@@ -43,7 +43,7 @@ public MantisPropertiesService getPropertiesService() {
     private MantisPropertiesLoader loadMantisPropertiesService() {
         MantisPropertiesLoader mpl = new DefaultMantisPropertiesLoader(new Properties());
         try {
-            mpl = (MantisPropertiesLoader) Class.forName("com.netflix.mantis.common.properties.MantisFastPropertiesLoader").getConstructor(Properties.class)
+            mpl = (MantisPropertiesLoader) Class.forName("com.netflix.io.mantisrx.shaded.common.properties.MantisFastPropertiesLoader").getConstructor(Properties.class)
```

Comment:
This is a new file. Not clear what's the right path here: `com.netflix.mantis` vs `com.netflix.io.mantisrx` vs `io.mantisrx`. Can you explain?

Reply:
cc @codyrioux

Reply:
I've never seen this but I think in general for the OSS stuff we'll want to stick to `io.mantisrx` as the root namespace.

Reply:
Reverting this file. This creeped in from find/replace. This class only exists in our internal repo and is loaded dynamically. We will not shade that class.